python/test/gluon/test_frontend.py

```diff
@@ -1661,46 +1661,38 @@ def test_amd_mfma_layout(target):
 
 
 @gluon.jit
-def add_int(a, b):
-    return a + b
+def amd_wmma_layout_kernel():
+    ttgl.full([64, 64], 0, ttgl.float16, layout=amd_layouts.AMDWMMALayout(version=2, transposed=True,
+                                                                          warps_per_cta=[1, 4]))
+    ttgl.full([64, 64], 0, ttgl.float16, layout=amd_layouts.AMDWMMALayout(version=2, transposed=True,
+                                                                          warps_per_cta=[2, 2]))
+    ttgl.full([64, 64], 0, ttgl.float16, layout=amd_layouts.AMDWMMALayout(version=2, transposed=False,
+                                                                          warps_per_cta=[1, 4]))
+    ttgl.full([64, 64], 0, ttgl.float16, layout=amd_layouts.AMDWMMALayout(version=2, transposed=False,
+                                                                          warps_per_cta=[2, 2]))
 
 
-@gluon.jit
-def infer_layout_for_amd_mfma_kernel():
-    layout: ttgl.constexpr = amd_layouts.AMDMFMALayout(version=3, instr_shape=[32, 32], transposed=True,
-                                                       warps_per_cta=[4,
-                                                                      1], elem_type=ttgl.int32, tiles_per_warp=[1, 1],
-                                                       ctas_per_cga=[1, 1], cta_split_num=[1, 1], cta_order=[1, 0])
-    a = ttgl.full([128, 32], 1, ttgl.int32, layout)
-    b = ttgl.reduce(a, 1, add_int)
-    ttgl.static_assert(b.type.layout == ttgl.SliceLayout(1, layout))
```

Comment:
Why remove this test? You're also missing a layout inference test for WMMA

Reply:
I think this test should already be covered in test_lowering.py::test_reduce_layouts. Do you still want to keep it?

Reply:
The point of the test is the static_assert which is not present in `test_reduce_layouts`, so no. Perhaps it would make sense to add the assert though.